addons/stock/report/report_deliveryslip.xml

```diff
@@ -119,7 +127,21 @@
                                 <!-- If not printing lots/serial numbers => merge lines with same product -->
                                 <t t-else="" name="aggregated_move_lines">
                                     <t t-set="aggregated_lines" t-value="o.move_line_ids._get_aggregated_product_quantities()"/>
-                                        <t t-call="stock.stock_report_delivery_aggregated_move_lines"/>
+                                    <t t-call="stock.stock_report_delivery_aggregated_move_lines"/>
+                                    <t t-set="empty_lines" t-value="o.move_lines.filtered(lambda x: x.product_uom_qty and x.quantity_done == 0)"/>
+                                    <tr t-foreach="empty_lines" t-as="move">
+                                        <td>
+                                            <span t-field="move.product_id"/>
+                                            <p t-if="move.description_picking != move.product_id.name">
```

Comment:
`and move_line.product_id.display_name != move.description_picking` to keep the same logic than `_get_aggregated_product_quantities`

Reply:
Done, at line 135 but also at lines 47 and 168.